examples/protocol_specification_ex/http.yaml

```diff
@@ -0,0 +1,20 @@
+---
+name: http
+author: fetchai
+version: 0.1.0
+license: Apache-2.0
+description: 'A protocol for HTTP requests and responses.'
+speech_acts:
```

Comment:
Following https://developer.mozilla.org/en-US/docs/Web/HTTP/Messages for design of the content fields